nitrogen/src/display.rs

```diff
@@ -17,6 +17,27 @@ use crate::resources::semaphore_pool::{SemaphoreList, SemaphorePool};
 use crate::submit_group::ResourceList;
 use std;
 
+pub struct SwapchainSetupConfig {
+    pub(crate) extent: gfx::window::Extent2D,
+    pub(crate) image_usage: gfx::image::Usage,
+    pub(crate) image_layers: gfx::image::Layer,
+    pub(crate) present_mode: gfx::PresentMode,
```

Comment:
All of these should probably be `pub` instead, otherwise other applications would be unable to actually configure anything :smile: